django-stubs/contrib/admin/options.pyi

```diff
@@ -190,16 +190,16 @@ class ModelAdmin(BaseModelAdmin[_ModelT]):
     def media(self) -> Media: ...
     def get_model_perms(self, request: HttpRequest) -> Dict[str, bool]: ...
     def get_form(
-        self, request: Any, obj: Optional[_ModelT] = ..., change: bool = ..., **kwargs: Any
+        self, request: HttpRequest, obj: Optional[_ModelT] = ..., change: bool = ..., **kwargs: Any
```

Comment:
For all the Django admin changes I read the source, `request` seems consistently used for `HttpRequest` throughout, as you'd hope